content/o11y/openshift-logging-lokistack/index.md

```diff
@@ -0,0 +1,571 @@
+---
+date: '2021-07-27'
+title: OpenShift Logging
+tags: ["Observability", "OCP"]
+authors:
+  - Andy Repton
+---
+
+A guide to shipping logs and metrics on OpenShift using the new LokiStack setup
+
+## Prerequisites
+
+1. OpenShift CLI (oc)
+1. Rights to install operators on the cluster
+1. Access to create S3 buckets (AWS/ROSA), Blob Storage Container (Azure), Storage Bucket (GCP)
+
+## For ROSA
+
+Create environment variables to use later in this process by running the following commands:
+
+```bash
+$ export REGION=$(oc get infrastructure cluster -o=jsonpath="{.status.platformStatus.aws.region}")
```

Comment:
all of these code blocks need to be indented, or they break the ordered list and it goes back to 1.